lib/docusign_rest/version.rb

```diff
@@ -1,3 +1,3 @@
 module DocusignRest
-  VERSION = "0.1.1"
+  VERSION = "0.1.2"
```

Comment:
I'd prefer to bump this when cutting a release to rubygems instead of here in the PR.